modules/amf-registration/amf-registration-service/service.xml

```diff
@@ -24,37 +24,33 @@
 
 		<!-- Other fields -->
 
-		<column name="first_name" type="String" />
-		<column name="last_name" type="String" />
-		<column name="email_address" type="String" />
-		<column name="user_name" type="String" />
-		<column name="male" type="String" />
+		<column name="firstName" type="String" />
+		<column name="lastName" type="String" />
+		<column name="emailAddress" type="String" />
+		<column name="userName_" type="String" />
+		<column name="gender" type="String" />
 		<column name="birthday" type="Date" />
 		<column name="password1" type="String" />
 		<column name="password2" type="String" />
-		<column name="home_phone" type="int" />
-		<column name="mobile_phone" type="int" />
+		<column name="homePhone" type="int" />
+		<column name="mobilePhone" type="int" />
 		<column name="address" type="String" />
 		<column name="address2" type="String" />
 		<column name="city" type="String" />
 		<column name="state" type="String" />
 		<column name="zip" type="String" />
-		<column name="security_question" type="String" />
-		<column name="security_answer" type="String" />
+		<column name="securityQuestion" type="String" />
+		<column name="securityAnswer" type="String" />
 		<column name="accepted_tou" type="boolean" />
```

Comment:
now I see you already some of the field names. next time, please add commit exclusively for defining the `entities` into service.xml and other one for building the service (i.e. including the autogenerated files). it makes easier the review process and keeps the commit history cleaner.